conf/genomes.config

```diff
@@ -22,6 +22,8 @@ params {
             known_dbsnp         = ""
             known_dbsnp_tbi     = ""
             ml_model            = "${params.local_genomes}/SentieonDNAscopeModel1.0.model"
+            reduced_penetrance  = "${params.local_genomes}/grch37_cust003-cmms-red-pen_-2017-.tsv"
+            score_config        = "${params.local_genomes}/rank_model_-v1.31-.ini"
```

Comment:
What do you think of supplying an example of the score config in assets. Could include what we put in as default annotations in the pipeline 

We also use a separate score configs for SNV/INDEL and SV